src/utils/evm.ts

```diff
@@ -13,7 +13,23 @@ export const chainNameToChainId: { [chainName: string]: number } = {
   fantom: 250,
 };
 
+export const chainNameToPublicRPCUrl: { [chainName: string]: string } = {
```

Comment:
I got these from here: https://rpc.info/ . These seem to be working fine but if we have others for testnets we can use you can recommend

Reply:
I like that we have these URLs for convenience and I appreciate you thinking like this, but I had forgotten how we do things in Airnode for contract deployment, where we have URLs for RPCs specified in the hardhat config file. It's a really clean way of doing things:
https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/credentials.example.json
https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/hardhat.config.js

We don't have to do it exactly like this, but perhaps have a credentials file instead which is pre-populated with RPC URLs. This will impact the choose-integration style flow.

Reply:
I've done something similar, can you take a look ?